test/wasm/testBinary4.js

```diff
@@ -4,4 +4,4 @@
 //-------------------------------------------------------------------------------------------------------
 
 var a = WScript.LoadWasmFile('test4.wasm', true);
-print(a());
+print(a(5));
```

Comment:
The original was in error. All functions in test4.wasm except the first take parameters.